dotcom-rendering/src/components/Ad.amp.tsx

```diff
@@ -98,6 +98,8 @@ export const Ad = ({
 		adType,
 	);
 
+	const refreshValue = id === 'ad-1' || id === 'ad-2' ? 'false' : '30';
```

Comment:
Non-blocking, but could add a comment here explaining why we vary the refreshing behaviour for the first two slots.

Reply:
Ah good shout, will do

Reply:
Thanks!